testapp/testapp/testmain/models.py

```diff
@@ -0,0 +1,22 @@
+# Create your models here.
+from decimal import Decimal
+
+from payments import PurchasedItem
+from payments.models import BasePayment
+
+
+class Payment(BasePayment):
+    def get_failure_url(self):
```

Comment:
This should return the URL to a view where a user is redirected after a failed payment (e.g.: `http://localhost:8000/payment-failed`).